### PR TITLE
Prepare for publishing

### PR DESCRIPTION
**Pull request by ennru**
Sort out a few features in the Makefile to get ready for publishing to TechHub.

References #264

### CODE REVIEW

File: docs-source/author-mode-site.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 site:
   title: "Akka Platform Guide"
-  url: https://akka.io/platform-guide
+  url: https://developer.lightbend.com/docs/akka-platform-guide
```
Comment (ignasi35) [inline docs-source/author-mode-site.yml:3]: Can we make the url in author mode to be `localhost'?
I have no idea, but I guess some links in local mode would better link locally?

**Review by ignasi35 — APPROVED** [review]
LGTM

File: Makefile [hunk 2]
```diff
@@ -39,6 +39,9 @@ html: clean
 		docs-source/site.yml
 	@echo "Done"
 
+html-staged: ${staging_dir} html
+	cp -r target/staging/snapshot/* ${staging_dir}
```
Comment (ignasi35) [inline Makefile:43]: We now have to `staging`'s: one is the variable and the other is the actual `staging/` directory. It makes things confusing: `cp -r target/staging/snapshot/* ${staging_dir}` --> I am copying staging into staging? 

Maybe we can use:

```
work_dir := ${ROOT_DIR}/target
staging_dir := ${work_dir}/staging
build_dir := ${CURDIR}/build/akka-platform-guide/ 
```

and, later on:

```
 	@echo "Done file://${staging_dir}/snapshot/index.html" 
```

etc...

**Review by ignasi35 — CHANGES_REQUESTED** [review]
(sorry, I changed my mind)

minor comment re variable naming

**Review by patriknw — APPROVED** [review]
LGTM

**Review by ignasi35 — APPROVED** [review]
nice, simpler!

**Pull request merged**